modules/light-clients/07-tendermint/module.go

```diff
@@ -1,6 +1,53 @@
 package tendermint
```

Comment:
Was unsure if we wanted to make a separate package for the `module.go`? This file layout doesn't align with SDK as we removed the `types` directory as used by [traditional sdk modules](https://github.com/cosmos/cosmos-sdk/tree/main/x/gov), but newer sdk modules have a different [layout](https://github.com/cosmos/cosmos-sdk/tree/main/x/group). Now is probably the last chance to change things around without too much disruption. I don't have strong preferences

Reply:
i guess i dont see the concrete benefit of making a separate package for module.go, and i dont see it being involved in any circular imports, so i don't see why it needs to be in a separate package. i think whichever we we choose to go we should just be consistent.

Reply:
Agreed. I think for now it makes sense to just add the `module.go` file. We can always update in the future if we see a need to